source/fundamentals/crud/read-operations/change-streams.txt

```diff
@@ -288,3 +288,56 @@ output:
 
 For a list of options, see the `FullDocument <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/changestream/FullDocument.html>`__
 API documentation.
+
+Split Large Change Stream Events
+--------------------------------
+
+Starting in MongoDB 7.0, you can use a ``$changeStreamSplitLargeEvent``
+aggregation stage to split events that exceed 16 MB into smaller fragments.
+``$changeStreamSplitLargeEvent`` returns the fragments sequentially using the
+change stream cursor. Each fragment includes a ``SplitEvent`` object containing
+the following fields:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 35 65
+
+   * - Field
+     - Description
+
+   * - ``fragment``
+     - The index of the fragment, starting at 1
+
+   * - ``of``
+     - The total number of fragments in the event
+
+Enable the ``$changeStreamSplitLargeEvent`` aggregation on your change stream as
```

Comment:
Suggestion:

I think "enable the ... aggregation" might not be an accurate term to describe setting up the change stream. I think the "modify change stream output" or "control change stream output" could work better as described in the [Server manual docs](https://www.mongodb.com/docs/manual/changeStreams/#modify-change-stream-output)

E.g.
The following example modifies your change stream to split large events: